lib/awful/completion.lua

```diff
@@ -90,7 +90,7 @@ function completion.shell(command, cur_pos, ncomp, shell)
         i = i + 1
     end
 
-    if cword_index == 1 and not string.find(words[cword_index], "/") then
+    if cword_index == 1 then
```

Comment:
5+ years later I honestly don't know what was the reason for that patch. What @blueyed says might be right ;)